<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QMainWindow" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomIn"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/new.png</normaloff>:/assets/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New text file</string>
   </property>
   <property name="statusTip">
    <string>Create a new text file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/open.png</normaloff>:/assets/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="statusTip">
    <string>Open an existing text file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/save.png</normaloff>:/assets/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="statusTip">
    <string>Save the text file to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/save_as.png</normaloff>:/assets/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save file as</string>
   </property>
   <property name="statusTip">
    <string>Save the text file under a new name</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/exit.png</normaloff>:/assets/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Editor</string>
   </property>
   <property name="statusTip">
    <string>Exit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/copy.png</normaloff>:/assets/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy selected text to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/cut.png</normaloff>:/assets/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="statusTip">
    <string>Cut selected text to clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/paste.png</normaloff>:/assets/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste text from clipboard to cursor position</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/edit_undo.png</normaloff>:/assets/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Undo changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/edit_redo.png</normaloff>:/assets/images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Redo changes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/zoom_out.png</normaloff>:/assets/images/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="statusTip">
    <string>Zoom out text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/zoom_in.png</normaloff>:/assets/images/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="statusTip">
    <string>Zoom in text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/info.png</normaloff>:/assets/images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Editor</string>
   </property>
   <property name="statusTip">
    <string>Show info about Editor</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="Editor.qrc">
     <normaloff>:/assets/images/run.png</normaloff>:/assets/images/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="statusTip">
    <string>Run program</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Editor.qrc"/>
 </resources>
 <connections/>
</ui>
